app/javascript/events.js

```diff
@@ -144,7 +145,7 @@ function initializeEvents() {
     const loadingMessage = document.getElementById("loading-message");
     
     // Show the loading message and clear previous content
-    loadingMessage.style.display = "block";
+    // loadingMessage.style.display = "block";
```

Comment:
Are we removing the loading messages? If so, there's no reason to keep it as a comment

Reply:
i dont remember commenting out this, maybe a mistake, i can add it in again

Reply:
Decide on whether we want this functionality or not and make corresponding changes